script/elastic_search/create_es_index_mappings.sh

```diff
@@ -0,0 +1,174 @@
+curl -XDELETE "https://vpc-mylibrarynyc-development-yvrqkaicwhwb5tiz3n365a3xza.us-east-1.es.amazonaws.com/teacherset"
```

Comment:
Same here about the host hardcoding.  Maybe it would be good to take this out into the encoded application params section.

Reply:
@daryanypl all hosts will be encode at a time. Stephen created user story in current sprint.